src/plugins/auto_batch/tests/functional/behavior/plugin/auto_batching_tests.cpp

```diff
@@ -53,7 +53,7 @@ INSTANTIATE_TEST_SUITE_P(smoke_AutoBatching_test_uint,
                          ::testing::Combine(::testing::Values(std::string(CommonTestUtils::DEVICE_BATCH) + ":" +
                                                               CommonTestUtils::DEVICE_TEMPLATE),
                                             ::testing::Values(DefaultParameter{ov::auto_batch_timeout.name(),
-                                                                               InferenceEngine::Parameter{1000}})),
+                                                                               InferenceEngine::Parameter{uint32_t(1000)}})),
```

Comment:
```suggestion
                                                                               ov::Any{uint32_t(1000)}})),
```

Reply:
for now, the functional test is from 1.0. So I think this kind of change can be put in another PR only for functional test